common/solar_system_initializers/hfe_initializer.txt

```diff
@@ -69,28 +75,58 @@ hfe_initializer_egypt_main = {
 			max = 270
 		}
 		init_effect = {
+			save_global_event_target_as = hfe_egypt_home_of_light
 			create_country = {
 				name = "Ju Ahet"
 				type = human_fallen_empire
 				name_list = cybrxkhan_fantasy_egyptian
-				ship_prefix = ""
+				ship_prefix = "Ra"
 				ignore_initial_colony_error = yes
 				species = event_target:hfe_egypt_species
+				authority = auth_imperial
 				ethos = {
 					ethic = ethic_fanatic_spiritualist
+					ethic = ethic_authoritarian
```

Comment:
Intended to have only 2 ethic points at start

Reply:
Not possible. They MUST have Authoritarian or Xenophobe in order to have slaves. Blame Paradox. When they wake up, they become Xenophobes. Besides, Authoritarianism is VERY much in character for the Ju Ahets!

Reply:
Do i really need to make screen from my branch to show them having slaves?

Reply:
Listen to me. The changes made to the Mod and the improvement and expansion of the handling of Ahet slaves neccesitate the alteration. Not only does it make them more unique and in-character, it has a story and mechanic purpsoe. Primarily, it fixes the bug where an Egalitarian and Xenophile nation that bans slavery becomes a slave state when it captures the Ju Ahet's enslaved Ahets.

This is not your branch. I made a lot of improvements and flavor to the Ahet slavery system and this was part of it.

Reply:
Then weights should be adjucted for Egalitarian ethic.

Reply:
How can it be set so that Pirates never become Egalitarians?

Reply:
Add weight for egalitarian, but not pirate

Reply:
I don't understand.

Reply:
You can allow any ethic for pirates and add weights for ai in species right so egalitarian non pirate will never use slaves.